php_apcu/datadog_checks/php_apcu/check.py

```diff
@@ -0,0 +1,29 @@
+import requests
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+
+class PhpApcuCheck(AgentCheck):
+    def check(self, instance):
+        url = instance.get('url')
+        if not url:
+            raise ConfigurationError('require url.')
+
+        self.log.debug('apcu check url[%s]', url)
+
+        try:
+            response = requests.get(url)
```

Comment:
The Agent base package support a request wrapper that would support additional options when making calls like timeout, proxies, tls verification, etc.

It's quite easy to set up, you just have to replace `request.get` with `self.http.get`. 

You can read more about it here: https://datadoghq.dev/integrations-core/base/http/

Reply:
fixed 7277ab326a70898fad0201f53ef010d6298e3d25